foreshadow/tests/test_feature_engineerers/test_feature_engineerers.py

```diff
@@ -0,0 +1,77 @@
+"""Test feature_engineerer.py"""
+import pandas as pd
+
+from foreshadow.core.column_sharer import ColumnSharer
+from foreshadow.feature_engineerers import (
+    FeatureEngineerer,
+    SmartFeatureEngineerer,
+)
+
+
+def test_feature_engineerer_fit():
+    """Test basic fit call."""
+    import numpy as np
+
+    data = pd.DataFrame(
+        {
+            "age": [10, 20, 33, 44],
+            "weights": [20, 30, 50, 60],
+            "financials": ["$1.00", "$550.01", "$1234", "$12353.3345"],
+        },
+        columns=["age", "weights", "financials"],
+    )
+    cs = ColumnSharer()
+    cs["domain"]["age"] = "personal"
+    cs["domain"]["weights"] = "personal"
+    cs["domain"]["financials"] = "financial"
+
+    cs["intent"]["age"] = "NumericIntent"
```

Comment:
It might make sense to name this as they are in the Intent PR so you don't have to change them later. As of right now I'm just calling them "Numeric", "Categoric", and "Text". I know this doesn't affect anything, just a suggestion :)